### ISSUE TITLE
Feature Engineering Checklist

### ISSUE THREAD

**Issue by rhiriyappa** [rhiriyappa/data-science-articles#10]
##TL;DR
**Feature engineering** is the process of using domain knowledge to extract features from raw data. These **features** can be used to improve the performance of machine learning algorithms. Feature engineering can be considered as applied machine learning itself. ([Wikipedia](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiBhPPUqPDvAhVRFzQIHTTzANcQFjAHegQICBAD&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FFeature_engineering&usg=AOvVaw2lrau707nYBhB3vCodb35Q))

## Active Link
https://rahulgupta1.medium.com/feature-engineering-checklist-7926ca3311b7
